src/tools/execution-results.h

```diff
@@ -69,18 +69,21 @@ struct ExecutionResults {
         auto* func = wasm.getFunction(exp->value);
         if (func->sig.results != Type::none) {
           // this has a result
-          Literal ret = run(func, wasm, instance);
+          Literals ret = run(func, wasm, instance);
           // We cannot compare funcrefs by name because function names can
           // change (after duplicate function elimination or roundtripping)
           // while the function contents are still the same
-          if (ret.type != Type::funcref) {
-            results[exp->name] = ret;
-            // ignore the result if we hit an unreachable and returned no value
-            if (results[exp->name].type.isConcrete()) {
-              std::cout << "[fuzz-exec] note result: " << exp->name << " => "
-                        << results[exp->name] << '\n';
+          for (auto& val : ret) {
+            if (val.type == Type::funcref) {
+              return;
```

Comment:
this early return avoids printing out anything for non-funcrefs. is that intended?

Reply:
Yes, although I could be convinced to change it. I'm not sure it's useful to perform a partial comparison of results.

Reply:
Not likely relevant in the near future, but if a function returns a funcref and an i32, why not compare the i32s?

Reply:
Ok, instead of bailing, I'll set all funcrefs to have the same name.